tests/api/conftest.py

```diff
@@ -38,23 +38,28 @@ def post(self, data=None, **kwargs):
         """
         if data:
             data = json.dumps(data, cls=DjangoJSONEncoder)
-        kwargs['content_type'] = 'application/json'
+        kwargs["content_type"] = "application/json"
         return super().post(API_PATH, data, **kwargs)
 
     def post_graphql(
-            self, query, variables=None, permissions=None,
-            check_no_permissions=True, **kwargs):
+        self,
+        query,
+        variables=None,
+        permissions=None,
+        check_no_permissions=True,
+        **kwargs,
```

Comment:
The ```**kwargs,)``` is not supported in python35 and is fixed in https://bugs.python.org/issue9232 for python36.

I'm not sure if this is a bug with black since ```target_version = ['py35', 'py36', 'py37', 'py38']``` is already set. I think black should take py35 as the minimal?

Reply:
Right, that's not good. According to docs formatted code should be valid with all provided target versions.

Reply:
Yep, we have to make sure that the code outputted from Black works under 3.5. I don't know what might be the issue though.

Reply:
Its a bug https://github.com/ambv/black/issues/759 :/